src/roller.js

```diff
@@ -174,13 +180,24 @@ class LMRTFYRoller extends Application {
     _onAbilityCheck(event) {
         event.preventDefault();
         const ability = event.currentTarget.dataset.ability;
-        this._makeRoll(event, 'rollAbilityTest', ability);
+        
+        if(game.system.id=="pf2e")  {
+            this._makeRoll(event, 'rollAbility', ability);
+        } else {
+            this._makeRoll(event, 'rollAbilityTest', ability);
+        }
     }
 
     _onAbilitySave(event) {
         event.preventDefault();
-        const ability = event.currentTarget.dataset.ability;
-        this._makeRoll(event, 'rollAbilitySave', ability);
+        const saves = event.currentTarget.dataset.ability;
```

Comment:
Might be good to also have it use `LMRTFY.SaveRollMethod` for example, and have it get set in the ready hook along with those abilities/saves/skills configs. Mostly so the code here remains simpler, especially if more systems are to be added, I don't want to see huge if/else if/else/etc...

Reply:
Done.